{
    "config": {
        "abort": {
            "already_configured": "Il servizio \u00e8 gi\u00e0 configurato",
            "cannot_connect": "Impossibile connettersi al server",
            "unknown": "\u00c8 stato generato un errore sconosciuto da python-opendata-transport"
        },
        "error": {
            "cannot_connect": "Impossibile connettersi al server",
            "unknown": "\u00c8 stato generato un errore sconosciuto da python-opendata-transport"
        },
        "step": {
            "user": {
                "data": {
                    "from": "Stazione di partenza",
                    "to": "Stazione di arrivo"
                },
                "title": "Trasporto pubblico svizzero"
            }
        }
    },
    "issues": {
        "deprecated_yaml_import_issue_bad_config": {
            "title": "La richiesta di importazione della configurazione YAML del trasporto pubblico svizzero non \u00e8 riuscita a causa di una configurazione errata"
        },
        "deprecated_yaml_import_issue_cannot_connect": {
            "title": "L'importazione della configurazione YAML del trasporto pubblico svizzero non pu\u00f2 connettersi al server"
        },
        "deprecated_yaml_import_issue_unknown": {
            "description": "La configurazione del trasporto pubblico svizzero tramite YAML \u00e8 in fase di rimozione ma si \u00e8 verificato un errore sconosciuto durante l'importazione della configurazione YAML. \n\n Controlla la tua configurazione o dai un'occhiata alla documentazione dell'integrazione.",
            "title": "L'importazione della configurazione YAML del trasporto pubblico svizzero non \u00e8 riuscita con un errore sconosciuto generato da python-opendata-transport"
        }
    }
}